resources/hu/cemu.po

```diff
@@ -545,77 +678,81 @@ msgstr ""
 "A választható nyelvek a nyelvi könyvtárban találhatóak\n"
 "A választás alkalmazásához újraindítás szükséges"
 
-#: GeneralSettings2.cpp:92
+#: GeneralSettings2.cpp
 msgid "Remember main window position"
 msgstr "Főablak pozíciójának megjegyzése"
 
-#: GeneralSettings2.cpp:94
+#: GeneralSettings2.cpp
 msgid "Restores the last known window position and size when starting Cemu"
 msgstr ""
 "Visszaállítja az ablak legutóbbi helyzetét és méretét a Cemu indításakor"
 
-#: GeneralSettings2.cpp:97
+#: GeneralSettings2.cpp
 msgid "Remember pad window position"
 msgstr "Mellékablak pozíciójának megjegyzése"
 
-#: GeneralSettings2.cpp:99
+#: GeneralSettings2.cpp
 msgid "Restores the last known pad window position and size when opening it"
 msgstr ""
 "Visszaállítja a mellékablak legutóbbi helyzetét és méretét a megnyitásakor"
 
-#: GeneralSettings2.cpp:103
+#: GeneralSettings2.cpp
 msgid "Discord Presence"
 msgstr "Discord Presence"
 
-#: GeneralSettings2.cpp:105
+#: GeneralSettings2.cpp
 msgid ""
 "Enables the Discord Rich Presence feature\n"
 "You will also need to enable it in the Discord settings itself!"
 msgstr ""
 "Engedélyezi a Discord Rich Presence-funkcióját\n"
 "Ezt a Discord beállításaiban is be kell kapcsolnod!"
 
-#: GeneralSettings2.cpp:110
+#: GeneralSettings2.cpp
 msgid "Fullscreen menu bar"
 msgstr "Teljes képernyős menüsor"
 
-#: GeneralSettings2.cpp:112
+#: GeneralSettings2.cpp
 msgid ""
 "Displays the menu bar when Cemu is running in fullscreen mode and the mouse "
 "cursor is moved to the top"
 msgstr ""
 "Megjeleníti a menüsort, ha teljes képernyős módban az egérmutatót feltolod a "
 "képernyő tetejére"
 
-#: GeneralSettings2.cpp:115
+#: GeneralSettings2.cpp
 msgid "Automatically check for updates"
 msgstr "Frissítések automatikus ellenőrzése"
 
-#: GeneralSettings2.cpp:117
+#: GeneralSettings2.cpp
 msgid "Automatically checks for new cemu versions on startup"
 msgstr ""
 "Induláskor a program automatikusan ellenőrzi, hogy van-e elérhető frissítés"
 
-#: GeneralSettings2.cpp:122
+#: GeneralSettings2.cpp
 msgid "Save screenshot"
 msgstr "Képernyőmentés rögzítése"
 
-#: GeneralSettings2.cpp:124
-msgid "Saves a screenshot directly to a file"
-msgstr "Egy fájlba menti a képernyőmentést"
+#: GeneralSettings2.cpp
+msgid ""
+"Pressing the screenshot key (F12) will save a screenshot directly to the "
+"screenshots folder"
+msgstr ""
+"A képernyőmentés gomb (F12) lenyomása egy képernyőmentést helyez a "
+"képernyőmentések mappába"
```

Comment:
The meaning of the original text is a bit different: "screenshots folder" refers to a folder named "screenshots", not to a generic folder containing screenshots